### PR TITLE
* supporting unicode strings for database responses vs. str

### PR DESCRIPTION
**Pull request by erichulser**

### CODE REVIEW

File: orb/core/column_types/data.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def dbRestore(self, db_value, context=None):
         """
         if db_value is not None:
             try:
-                return pickle.loads(db_value)
+                return pickle.loads(str(db_value))
```
Comment (coolnay309) [inline orb/core/column_types/data.py]: what does this line do ?

File: orb/core/connection_types/sql/postgres/psqlconnection.py [hunk 2]
```diff
@@ -16,17 +16,24 @@
...
     import psycopg2 as pg
+    import psycopg2.extensions as pg_ext
+
     from psycopg2.extras import DictCursor, register_hstore, register_json
-    from psycopg2.extensions import QueryCanceledError
 
 except ImportError:
     log.debug('For PostgreSQL backend, download the psycopg2 module')
 
-    QueryCanceledError = orb.errors.DatabaseError
```
Comment (coolnay309) [inline orb/core/connection_types/sql/postgres/psqlconnection.py]: not needed anymore ?

File: orb/core/connection_types/sql/postgres/psqlconnection.py [hunk 3]
```diff
@@ -89,7 +96,7 @@ def _execute(self,
             rowcount = cursor.rowcount
 
         # look for a cancelled query
-        except QueryCanceledError as cancelled:
+        except pg_ext.QueryCanceledError as cancelled:
```
Comment (coolnay309) [inline orb/core/connection_types/sql/postgres/psqlconnection.py]: I see!

File: orb/core/column_types/data.py (hunk 1 above)
Comment (erichulser) [inline orb/core/column_types/data.py]: a cPickle is a kind of python data construct, one of the options for saving/loading DB data.  we don't use it...but it's available.  apparently pickle.loads requires a str

File: orb/core/connection_types/sql/postgres/psqlconnection.py (hunk 2 above)
Comment (erichulser) [inline orb/core/connection_types/sql/postgres/psqlconnection.py]: used with the module import now `pg_ext.QueryCanceledError`.  It doesn't need to be explicitly defined, I only ever had it in there as a linter satisfaction, one of those "QueryCanceledError could be not defined in XYZ scenario" complaints

**Pull request merged**